server/storage/interface.go

```diff
@@ -36,6 +36,11 @@ type MetaStore interface {
 	// not found, it returns storage.ErrNotFound
 	GetChecksum(gun, tufRole, checksum string) (created *time.Time, data []byte, err error)
 
+	// GetVersion returns the given TUF role file and creation date for the
+	// GUN with the provided version. If the given (gun, role, version) are
+	// not found, it returns storage.ErrNotFound
+	GetVersion(gun, tufRole, version string) (created *time.Time, data []byte, err error)
```

Comment:
I wouldn't block the PR on this, as it's pretty minor and and can easily be fixed later, but my general preference for interfaces to be specific as to the types of arguments expected;  the version is defined as an int in the TUF data, etc., and this would let the compiler verify the type instead of getting an input error when it fails to convert.